CHANGELOG.md

```diff
@@ -8,8 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Stabilize `call_runtime` ‒ [#1749](https://github.com/paritytech/ink/pull/1749)
 
 ### Added
+- Persist `Environment` in metadata - [#1741](https://github.com/paritytech/ink/pull/1741)
```

Comment:
This one has already been released in `4.2.0` https://github.com/paritytech/ink/blob/master/CHANGELOG.md#added-1. Please remove it

Reply:
done

Reply:
It is still there 
```suggestion
```

Reply:
apologies, I removed the line for this in the 4.2 release section rather than the unreleased section. Have fixed it now